cvmfs/catalog_mgr_ro.h

```diff
@@ -66,7 +64,18 @@ class SimpleCatalogManager : public AbstractCatalogManager<Catalog> {
     return (relative_path == "") ? "" : "/" + relative_path;
   }
 
+  bool useLocalCache() { return !local_cache_dir_.empty(); }
+
+  std::string                local_cache_dir_;  // absolute path to local cache
```

Comment:
Consistent naming in this class:
```suggestion
  std::string                dir_cache_;  // absolute path to local cache
```

Reply:
i would prefer either `local_cache_dir_` or `cache_dir_`

Reply:
renamed it to `dir_cache_` to keep file consistency